### PR TITLE
Deprecate flutter performance dashboard

### PR DESCRIPTION
**Pull request by keyonghan**
This is first step to skip performance dashboard build, and drop it from the flutter dashboard.

Part of https://github.com/flutter/flutter/issues/76235.

### CODE REVIEW

File: app_flutter/lib/logic/links.dart [hunk 1]
```diff
@@ -25,11 +25,6 @@ List<CocoonLink> createCocoonLinks(BuildContext context) {
       icon: const Icon(Icons.build),
       action: () => Navigator.pushReplacementNamed(context, BuildDashboardPage.routeName),
     ),
-    CocoonLink(
-      name: 'Benchmarks',
-      icon: const Icon(Icons.show_chart),
-      action: () => launch('/benchmarks.html'),
-    ),
     CocoonLink(
       name: 'Framework Benchmarks on Skia Perf',
```
Comment (CaseyHillers) [inline app_flutter/lib/logic/links.dart:29]: You can drop the skia perf wording now (eg "Framework Benchmarks")

File: app_flutter/test/widgets/commit_box_test.dart [hunk 2]
```diff
@@ -73,7 +73,7 @@ void main() {
 
     // Since the overlay positions itself a little below the center of the widget,
     // it is safe to click the center of the widget to close it again.
-    await tester.tap(find.byType(CommitBox));
+    await tester.tap(find.byType(CommitBox), warnIfMissed: false);
```
Comment (CaseyHillers) [inline app_flutter/test/widgets/commit_box_test.dart:76]: Why was this added?

**Review by CaseyHillers — APPROVED** [review]
LGTM when comments are addressed.

You should schedule an announcement to the team so there are no surprises.

File: app_flutter/lib/logic/links.dart (hunk 1 above)
Comment (keyonghan) [reply app_flutter/lib/logic/links.dart:29]: Done.

> LGTM when comments are addressed.
> 
> You should schedule an announcement to the team so there are no surprises.

A notice email will be sent out by today to schedule the deprecation by end of tomorrow.
This PR will not be submitted today.

File: app_flutter/test/widgets/commit_box_test.dart (hunk 2 above)
Comment (keyonghan) [reply app_flutter/test/widgets/commit_box_test.dart:76]: Noticed below warming message in the test:
```

Warning: A call to tap() with finder "exactly one widget with type "CommitBox" (ignoring offstage widgets): CommitBox(dependencies: [MediaQuery], state: _CommitBoxState#c28a8)" derived an Offset (Offset(400.0, 300.0)) that would not hit test on the specified widget.
Maybe the widget is actually off-screen, or another widget is obscuring it, or the widget cannot receive pointer events.
The finder corresponds to this RenderBox: RenderSemanticsAnnotations#dd01e
The hit test result at that offset is: HitTestResult(_RenderColoredBox#88ab2@Offset(400.0, 300.0), RenderConstrainedBox#d8a30@Offset(400.0, 300.0), RenderPointerListener#fca1c@Offset(400.0, 300.0), RenderSemanticsGestureHandler#ad194@Offset(400.0, 300.0), RenderStack#a2174@Offset(400.0, 300.0), _RenderTheatre#a633d@Offset(400.0, 300.0), RenderSemanticsAnnotations#63686@Offset(400.0, 300.0), RenderAbsorbPointer#6dc7b@Offset(400.0, 300.0), RenderPointerListener#10f13@Offset(400.0, 300.0), RenderCustomPaint#a5927@Offset(400.0, 300.0), RenderSemanticsAnnotations#94430@Offset(400.0, 300.0), RenderSemanticsAnnotations#2d92d@Offset(400.0, 300.0), HitTestEntry#b7433(RenderView#08885), HitTestEntry#2d8e6(<AutomatedTestWidgetsFlutterBinding>))
#0      WidgetController._getElementPoint (package:flutter_test/src/controller.dart:953:25)
#1      WidgetController.getCenter (package:flutter_test/src/controller.dart:836:12)
#2      WidgetController.tap (package:flutter_test/src/controller.dart:271:18)
#3      main.<anonymous closure> (file:///Users/keyonghan/Documents/work/go/src/cocoon/app_flutter/test/widgets/commit_box_test.dart:76:18)
<asynchronous suspension>
#4      StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart)
<asynchronous suspension>
To silence this warning, pass "warnIfMissed: false" to "tap()".
To make this warning fatal, set WidgetController.hitTestWarningShouldBeFatal to true.
```

Is this expected?

Comment (CaseyHillers) [reply app_flutter/test/widgets/commit_box_test.dart:76]: Do you have a recent version of Flutter (`flutter doctor -v` < 3 weeks old) and is this branch based on a recent revision from flutter/cocoon? Previous try jobs are not failing on this.

Comment (keyonghan) [reply app_flutter/test/widgets/commit_box_test.dart:76]: It's not failing, just a warming fix.

Comment (keyonghan) [reply app_flutter/test/widgets/commit_box_test.dart:76]: The warming has been there, see a previous run: https://ci.chromium.org/ui/p/flutter/builders/try/Cocoon/1678/overview
I can leave the warning as it is if you have concerns.

Comment (CaseyHillers) [reply app_flutter/test/widgets/commit_box_test.dart:76]: Thanks for pointing it out. I'd rather keep the warning so if this causes issues in the future we have a lead.

Comment (keyonghan) [reply app_flutter/test/widgets/commit_box_test.dart]: Change restored.
